qiskit_nature/problems/sampling/protein_folding/peptide/chains/main_chain.py

```diff
@@ -26,77 +26,64 @@ class MainChain(BaseChain):
 
     def __init__(
         self,
-        main_chain_len: int,
-        main_chain_residue_sequence: List[str],
-        side_chain_lens: List[int],
-        side_chain_residue_sequences: List[Optional[str]],
+        main_chain_residue_sequence: str,
```

Comment:
Should this also be a `List(str)` or not?

Reply:
No that is the proposal of #309: rather than deal with a list of single-character strings, a simple string itself is much more reasonable. This should also resolve your comment below.